@here/harp-mapview/lib/geometry/TileGeometryLoader.ts

```diff
@@ -286,6 +286,10 @@ export class SimpleTileGeometryLoader implements TileGeometryLoader {
             if (stats.enabled) {
                 const geometryCreationTime = PerformanceTimer.now() - now;
                 const currentFrame = stats.currentFrame;
+
+                // Account for the geometry creation in the current frame.
+                currentFrame.addValue("render.fullFrameTime", geometryCreationTime);
+
```

Comment:
I wonder how much details we would like to spot... but reasonably we would also need to add the time for processing techniques via `TileGeometryCreator`. You may find it in `prepareForRender()` method in the same class.

Reply:
Okay, I will check.